From 7b3dc4ea44fe83f98acd901870dda6a7f89ddfcd Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Thu, 19 Jul 2018 17:08:48 +0800
Subject: [PATCH 271/706] drm: hdp: Set DPTX debug to skip sleep

This patch sets DPTX debug related features to skip sleep.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 92ccdf855d77b0960c36039eba23833ff76c1654)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/imx-dp.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/imx/hdp/imx-dp.c b/drivers/gpu/drm/imx/hdp/imx-dp.c
index 8f36a2c87e86..ad309f90189a 100644
--- a/drivers/gpu/drm/imx/hdp/imx-dp.c
+++ b/drivers/gpu/drm/imx/hdp/imx-dp.c
@@ -119,6 +119,8 @@ void dp_mode_set(state_struct *state, int vic, int format, int color_depth, int
 	u8 eventId;
 	u8 HPDevents;
 
+	CDN_API_DPTX_SetDbg_blocking(state, DPTX_DBG_SET_PWR_SKIP_SLEEP);
+
 	ret = CDN_API_DPTX_SetHostCap_blocking(state,
 		max_link_rate,
 		(num_lanes & 0x7) | ((ssc & 1) << 3) | ((scrambler & 1) << 4),
-- 
2.17.1

